### PR TITLE
Fixes 200316

### PR DESCRIPTION
**Pull request by lyakh**
As suggested by @lgirdwood moving all fixes to a separate PR from #2513

### CODE REVIEW

File: tools/topology/m4/bytecontrol.m4 [hunk 1]
```diff
@@ -7,19 +7,19 @@ define(`CONTROLBYTES_MAX',
 `#$1'
 `	max STR($2)')
 
-dnl CONTROLMIXER_TLV(comment, value)
+dnl CONTROLBYTES_TLV(comment, value)
```
Comment (lgirdwood) [inline tools/topology/m4/bytecontrol.m4]: Can you say in the commit message what this fixes, it's not clear why you are making this change ?

File: src/schedule/edf_schedule.c [hunk 2]
```diff
@@ -22,6 +22,8 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#include <sof/lib/cpu.h>
```
Comment (lgirdwood) [inline src/schedule/edf_schedule.c]: Can you make this a new PR, there will be discussion on this that will block the others being merged.

File: tools/topology/m4/bytecontrol.m4 (hunk 1 above)
Comment (lyakh) [reply tools/topology/m4/bytecontrol.m4]: it seems pretty clear, just look at the next line:
```
define(`CONTROLBYTES_TLV',
```
it's a pure copy-paste comment correction, and the file name is also "bytecontrol.m4" as opposed to "mixercontrol.m4" where that copying comes from. But yes, I can add that to the commit message...

**Comment by lyakh** [comment]
@lgirdwood @tlauda any more comments to this one?

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lyakh** [comment]
@lgirdwood @tlauda any progress?

**Comment by lgirdwood** [comment]
CI showing know issues.

**Pull request merged**